Assets/MixedRealityToolkit/Services/MixedRealityToolkit.cs

```diff
@@ -167,11 +167,7 @@ public void ResetConfiguration(MixedRealityToolkitConfigurationProfile profile)
                 return false;
             }
 
-#if !UNITY_EDITOR
             if (!Application.platform.IsPlatformSupported(supportedPlatforms))
-#else
-            if (!EditorUserBuildSettings.activeBuildTarget.IsPlatformSupported(supportedPlatforms))
```

Comment:
The idea here was that you care more about the active build target in the editor than the fact that you're in the editor.

I added the editor as an explicit target to mark editor-only services, which did muddy the definition a little bit (for example, you can't currently mark something as in-app only, ignoring running in the editor).

Reply:
Removing this would make things a little less configurable, i think, since all services that *can* run in the editor *will* run in the editor, even if one only runs when the build target is Standalone and one only runs when the build target is UWP.

Reply:
I think I see what you mean but it's more confusing. If I mark something as supported in WindowsEditor, that should be the end all truth. 

I see what you mean in that if I mark a service as WindowsStandalone and WindowsEditor, it will still run in-editor with build target UWP or really any windows build target. But saying we only use BuildTarget in editor is not obvious compared to marking a service that yes I can run in editor